DEPENDENCIES

```diff
@@ -97,9 +97,9 @@ maven/mavencentral/io.swagger.core.v3/swagger-jaxrs2-jakarta/2.2.2, Apache-2.0,
 maven/mavencentral/io.swagger.core.v3/swagger-models-jakarta/2.2.2, Apache-2.0, approved, #5919
 maven/mavencentral/jakarta.activation/jakarta.activation-api/2.1.0, EPL-2.0 OR BSD-3-Clause OR GPL-2.0-only with Classpath-exception-2.0, approved, ee4j.jaf
 maven/mavencentral/jakarta.annotation/jakarta.annotation-api/2.1.1, EPL-2.0 OR GPL-2.0-only with Classpath-exception-2.0, approved, ee4j.ca
-maven/mavencentral/jakarta.inject/jakarta.inject-api/2.0.1, Apache-2.0, approved, clearlydefined
-maven/mavencentral/jakarta.json/jakarta.json-api/2.1.1, EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0, approved, #7907
-maven/mavencentral/jakarta.transaction/jakarta.transaction-api/2.0.0, EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0, approved, #7697
+maven/mavencentral/jakarta.inject/jakarta.inject-api/2.0.1, Apache-2.0, approved, ee4j.cdi
```

Comment:
source for LICENSE information changed, still approved.

Reply:
Hey, I would also suggest implementing the dependencies workflow to have an automated check on each PR like [here](https://github.com/eclipse-tractusx/portal-backend/blob/main/.github/workflows/dependencies.yaml). This would make sure that we don't merge PRs that have outdated DEPENDENCIES files.

Reply:
> Hey, I would also suggest implementing the dependencies workflow to have an automated check on each PR like [here](https://github.com/eclipse-tractusx/portal-backend/blob/main/.github/workflows/dependencies.yaml). This would make sure that we don't merge PRs that have outdated DEPENDENCIES files.

will do